cmd/nerdctl/image_pull_linux_test.go

```diff
@@ -145,3 +145,23 @@ CMD ["echo", "nerdctl-build-test-string"]
 	newKeyPair := newCosignKeyPair(t, "cosign-key-pair-test")
 	base.Cmd("pull", testImageRef, "--verify=cosign", "--cosign-key="+newKeyPair.publicKey).AssertFail()
 }
+
+func TestPullSoci(t *testing.T) {
+	//TODO: validate that mounts match the mounts of soci rpull
+	testutil.DockerIncompatible(t)
+	base := testutil.NewBase(t)
+	requiresSoci(base)
+
+	//calling nerdctl run with '--snapshotter=soci ...'
+	base.Cmd("--snapshotter=soci", "pull", "--rm", testutil.FfmpegSociImage).Run()
```

Comment:
Could you change to `err := base.Cmd("--snapshotter=soci", "pull", "--rm", testutil.FfmpegSociImage).Run()` and then log the error?